config/routes.rb

```diff
@@ -1,6 +1,138 @@
 Rails.application.routes.draw do
+  # HTTP Verb examples
+  #
+  # The word "Record" usually means an item in the database
+  #
+  # GET => Used to get information
+  # POST => Used to create new records
+  # PUT => Used to update existing records
+  # DELETE => Used to delete records
+
+  # All of these examples do the same thing, but
+  # using different HTTP verbs
+
+  # Only GET works from a browser URL
   get 'example1', to: "example#example1"
-  post 'example2', to: "example#example2"
-  put 'example3', to: "example#example3"
-  delete 'example4', to: 'example#example4'
+  put 'example1', to: "example#example1"
```

Comment:
@nw0428 Can you add the curl commands as comments to this so the students have a reference?